app/services/teachers/current_schools.rb

```diff
@@ -0,0 +1,14 @@
+module Teachers
+  class CurrentSchools
+    def initialize(trn:)
+      @trn = trn
+    end
+
+    def ect_at
+      teacher = Teacher.find_by(trn: @trn)
+      return nil unless teacher
+
+      teacher.ect_at_school_periods.max_by(&:created_at)&.school
```

Comment:
Also this will pull all the `ect_at_school_periods` and sort them in Ruby. An ECT can only have one _ongoing_ `ect_at_school_period` so this could be `teacher.ect_at_school_periods.ongoing.first.school`.

Reply:
You are right! I realised after I committed, I was going to circle back later but I've done it as part of this commit too: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/554/commits/c5adfa43159babdd439ad94fdbc5cc3cfccc467c

Thank you!